### ISSUE TITLE
Add action support to build

### ISSUE THREAD

**Issue by johnsca** [juju/charm-tools#30]
To support reactive actions, we should create boilerplate action executables based on the contents of `actions.yaml` during the `charm build` process.

**Comment by bcsaller** [comment]
Skeptical. Each file should generally be owned by one layer. This would be a generated file that should be hand edited. We could do this, but it is more of a template pattern than the current model we have.

**Comment by johnsca** [comment]
I don't think it should be hand edited at all.  It should have the exact same contents as the hooks generated by interface layers.  The custom code for the actions would live under `reactive/` like all other reactive code.
